shop/.dockerignore

```diff
@@ -9,3 +9,6 @@
 !composer.*
 !yarn.*
 !.stylelint*
+!tests/
+!setup_test
```

Comment:
I don't know. In this case, when you run codeception, it tries to look specifically for those two files.